### PR TITLE
Bugfix/console log

### PR DESCRIPTION
**Pull request by khackskjs**
1. now debug log always shows received false, I fixed it.
2. to prevent wrong error log with webpack HMR, in my case vue.js web app.
image

### CODE REVIEW

File: src/Bellhop.js [hunk 1]
```diff
@@ -115,7 +115,9 @@ export class Bellhop extends BellhopEventDispatcher {
       // Check to see if the data was sent as a stringified json
       if ('string' === typeof data) {
         try {
+          if (data && !data.startsWith('webpack')) {
```
Comment (902seanryan) [inline src/Bellhop.js:118]: For now I'm going to request that this additional check be removed. See comment for more info

**Review by 902seanryan — CHANGES_REQUESTED** [review]

**Comment by 902seanryan** [comment]
@khackskjs 
In regards to the `webpack` check. I think that check is just too specific to really require its inclusion. The error is technically correct since we're trying to parse a string rather than stringified JSON. Its just that the message is coming from Webpack not from within the Springroll ecosystem. 
The error itself also shouldn't be halting any execution since its just a `console.error` call. 
If the error message is a bigger problem, feel free to open up an issue and we can have a proper discussion on how best to handle it. 

Thanks again for catching bugs and presenting fixes. Always appreciated!

**Comment by khackskjs** [comment]
@902seanryan I understand what's your point and I agree that is' too specific case. as you said, it's just log without any collaption. so I suggest simply change log level with warn.
I believe error log is kind of unexpected situation or failed to work in business logic.

**Comment by deycorinne** [comment]
@902seanryan sounds good to me. let's do it!

File: src/Bellhop.js [hunk 2]
```diff
@@ -307,7 +309,7 @@ export class Bellhop extends BellhopEventDispatcher {
    */
   logDebugMessage(received = false, message) {
     if (this.debug && typeof this.debug === 'function') {
-      this.debug({ isChild: this.isChild, received: false, message: message });
+      this.debug({ isChild: this.isChild, received, message: message });
```
Comment (khackskjs) [inline src/Bellhop.js:312]: @902seanryan @deycorinne Please check this line also. 
`false` is suppsed to be removed.
thank you.

**Review by 902seanryan — APPROVED** [review]
@khackskjs Sorry could you point this one to develop as well?

**Comment by khackskjs** [comment]
Is there anything for me to do to merge blocked PR? 
Reason why I ask you it's my first time to merge blocked PR that I made a PR

**Pull request merged**